base/process.jl

```diff
@@ -81,7 +88,15 @@ end
 
 immutable DevNullStream <: AsyncStream end
 const DevNull = DevNullStream()
+isreadable(::DevNullStream) = false
+iswritable(::DevNullStream) = true
+isopen(::DevNullStream) = true
+read{T<:DevNullStream}(::T, args...) = throw(EOFErorr())
+write{T<:DevNullStream}(::T, args...) = 0
```

Comment:
This might be ok for now but I'm not sure it's right. Semantically, I believe all bytes are successfully written to DevNull.